docs/zh_CN/install.md

```diff
@@ -23,8 +23,11 @@ pip install -U paddlepaddle-gpu==2.0.0rc0
 </code></pre> </details> </td> <td align="left"><details><summary> install </summary><pre><code>python -m pip install https://paddle-wheel.bj.bcebos.com/2.0.0-rc0-gpu-cuda9-cudnn7-mkl%2Fpaddlepaddle_gpu-2.0.0rc0.post90-cp36-cp36m-linux_x86_64.whl
 </code></pre> </details> </td> </tr></tbody></table>
 
-### 2. 通过Pip安装
+支持更多系统的安装教程请前往[paddlepaddle官网](https://www.paddlepaddle.org.cn/install/quick)
```

Comment:
add same content for corresponding en readme

Reply:
english version install has been updated, and irrelevant file has been deleted.